requirements/quality-extra.txt

```diff
@@ -0,0 +1,2 @@
+pylint
+edx-lint                  # edX pylint rules and plugins
```

Comment:
So if quality.txt is a generated file, shouldn't it be excluded from committing into the repo?